apps/central/src/main.js

```diff
@@ -1,12 +1,12 @@
 import Vue from "vue";
 import App from "./App.vue";
 import VueRouter from "vue-router";
-import Groups from "./components/Groups";
-import Admin from "./components/admin/Admin";
-import ManageSettings from "./components/admin/ManageSettings";
-import ManageUsers from "./components/admin/ManageUsers";
+import Groups from "./components/Groups.vue";
+import Admin from "./components/admin/Admin.vue";
+import ManageSettings from "./components/admin/ManageSettings.vue";
+import ManageUsers from "./components/admin/ManageUsers.vue";
 
-import "molgenis-components/dist/style.css";
+import "../../molgenis-components/dist/style.css";
```

Comment:
this is bypassing the packaging system. Although I see pragmatism, my gut says: we should not do this because might result that packages might not properly pick up changes. Instead, the molgenis-components package should properly export the style.css (see the schema2 PR). I also expect we will want to publish molgenis-components to npm at some point and then this certainly doesn't work.

Similar discussions here https://github.com/vitejs/vite/discussions/2657

Reply:
understand but the whole setup is not tuned for npm publishing, gradle currently does lots of custom build/move/copy steps. i saw the schema2 pr but am not sure why this is needed as the css already seems to be part of the dist, i'll look in to changing the PR. However i am not sure about the solution as discussed in the vite comments.

Reply:
updated as requested